spec/requests/api/drops_controller_spec.rb

```diff
@@ -27,9 +29,46 @@
         parsed_response = ActiveSupport::JSON.decode(response.body)
         expect(parsed_response[0]["place"].keys).to eq(["id", "name", "latitude", "longitude"])
       end
+    end
+
+
+    describe "GET /api/drops" do
+
+      context 'with latitude and longitude given' do # will always set the radio to 10km
+        let!(:drop_in_berlin) { FactoryGirl.create :drop, :drop_in_rheinsberger}
+        let!(:drop_in_sydney) { FactoryGirl.create :drop, :drop_in_sydney}
```

Comment:
there's a short cut for Factory Girl, instead you can do:
`let!(:drop_in_berlin) { create :drop, :drop_in_rheinsberger }`

I *think* you can do this because we included this line in our rails helper, which gives us access to alternate (shorter) factory girl methods. BUT i'm not 100% sure about that so don't quote me on it!
https://github.com/sounddrop/sounddrop/blob/master/spec/rails_helper.rb#L55